### DISCUSSION TITLE
Watchface maker improvements

### DISCUSSION THREAD

**Discussion by kartun83** [zepp-health/zeppos-forum#221] [Developer Feedback]
Hello.

There is a [guideline](https://docs.zepp.com/docs/watchface/specification/#default-data) related to default data for generation of preview screen. IT would be great time saver to have functions:

- init all controls with random data or simulate timeflow
- init all controls with required data and force preview generation to use this function

**Comment by huamitech** [comment]
Thank you for your valuable suggestion. We have received it and will carefully consider it.
